packages/calypso-analytics/src/utils/is-region-in-ccpa-zone.ts

```diff
@@ -20,14 +20,17 @@ const CCPA_US_REGIONS = [
  * @param region The region to look for.
  * @returns Whether the region is in the GDPR zone
  */
-export default function isRegionInCcpaZone( countryCode?: string, region?: string ): boolean {
+export default function isRegionInCcpaZone(
+	countryCode: string | undefined,
+	region: string | undefined
+): boolean {
```

Comment:
The two parameters are not really optional, we always call the function with both. They can just be `undefined`. Making the type more precise here.